sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM rust:1.32 as base-build
+
+ENV TMP_BUILD_DIR=/tmp/build
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
+ENV RUST_VERSION=1.32.0
+ENV PATH="$CARGO_HOME/bin:$PATH"
+
+ARG FIRECRACKER_VERSION="0.15.0"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        curl \
+        make \
+        musl-tools \
+        libseccomp-dev \
+        btrfs-tools \
```

Comment:
I don't believe Firecracker requires btrfs-tools.

Reply:
it's a CRI dependency

Reply:
It doesn't ) I bet this is for `containerd`. @xibz how about using `no_btrfs` build tag instead?

Reply:
Ah cool! Didn't know that was a thing. Thanks, will update.

Reply:
`make` should accept `BUILDTAGS` parameter. https://github.com/containerd/cri/blob/eb27e526f5d23c4c762d25f80351c632efe4f7ab/hack/install/install-containerd.sh#L40
Can you try it?

Reply:
Done!

Reply:
This stage of the Dockerfile is not used to build CRI or containerd.